packages/create-turbo/turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../../docs/public/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "test": {
+      "dependsOn": ["build"]
+    }
```

Comment:
We don't have a `build` task available here.

Reply:
It should inherit the `build` task from root config! this is additive. This is the test that's intended to prove this case works: https://github.com/vercel/turbo/blob/main/cli/integration_tests/composable_config/composing-add-tasks.t

Reply:
<details>
  <summary>pnpm run turb run test --filter=create-turbo --graph</summary>

```
digraph {
	compound = "true"
	newrank = "true"
	subgraph "root" {
		"[root] ___ROOT___#build" -> "[root] ___ROOT___"
		"[root] create-turbo#build" -> "[root] tsconfig#build"
		"[root] create-turbo#test" -> "[root] create-turbo#build"
		"[root] create-turbo#test" -> "[root] tsconfig#build"
		"[root] tsconfig#build" -> "[root] ___ROOT___#build"
	}
}
```

</details>